code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -311,8 +311,12 @@
 
 /mob/living/simple_animal/gib()
 	if(butcher_results || guaranteed_butcher_results)
+		var/list/butcher
```

Comment:
```
var/list/butcher = list()
``` 
And drop the if's after this since the following is an allowed statement.
```
list += null
```

Reply:
Ill fix the first, but list += null adds the null to the list which is the issue we need to fix. Double checked with MoMMI to be sure.

Reply:
oh right